app/src/main/java/org/fossasia/openevent/general/LoginActivity.kt

```diff
@@ -70,4 +66,9 @@ class LoginActivity : AppCompatActivity() {
         super.onDestroy()
         compositeDisposable.dispose()
     }
+
+    private fun showProgress(enabled:Boolean) {
```

Comment:
Read kotlin style guideline, there should be space after : 

I'm accepting for now. Good job